src/components/RestaurantDropdown/RestaurantDropdown.js

```diff
@@ -61,6 +62,15 @@ const RestaurantDropdown = ({
   const DropdownToggle = Dropdown.Toggle;
   const DropdownMenu = Dropdown.Menu;
 
+  let lastVisited = [];
+  if (restaurant.all_decision_count > 0) {
```

Comment:
It’s probably less error-prone to check for the presence of `pastDecisions[restaurant.id]` here instead.